src/lib/masq.js

```diff
@@ -699,6 +715,25 @@ class Masq {
     this.syncApps()
   }
 
+  /*
+   * Set a watcher for the given key (e.g /devices, /profile)
+   * Apply handleUpdate for each update
+   *
+   * @param {string} key The key path
+   * @param {function} handleUpdate The function to apply for each update
+   * @returns The watcher
+   */
+  addWatcher (key, handleUpdate) {
+    const watcher = watch(this.profileDB, this.nonce, key, handleUpdate)
+    this.watchers.push(watcher)
+  }
+
+  destroyWatchers () {
+    this.watchers.forEach(watcher => {
+      if (watcher !== null || undefined) { watcher.destroy() }
```

Comment:
(watcher !== null || watcher !=== undefined)

Reply:
Or you can write if (!!watcher) { }

Reply:
ok done